src/common/tagging/external/tag_group.go

```diff
@@ -0,0 +1,43 @@
+package external
+
+import (
+	"io/ioutil"
+
+	"github.com/bridgecrewio/yor/src/common/logger"
+	"github.com/bridgecrewio/yor/src/common/structure"
+	"github.com/bridgecrewio/yor/src/common/tagging"
+	"github.com/bridgecrewio/yor/src/common/tagging/tags"
+	"gopkg.in/yaml.v2"
+)
+
+type TagGroup struct {
+	tagging.TagGroup
+	configFilePath string
+	config         map[interface{}]interface{}
+}
+
+func (t *TagGroup) InitConfigFile(configFilePath string) {
+	t.configFilePath = configFilePath
+}
+
+func (t *TagGroup) InitTagGroup(_ string, skippedTags []string) {
+	t.SkippedTags = skippedTags
+}
```

Comment:
@tronxd We should load the tags from the conf file here, right?

Reply:
Yes